Bottom: f874300721c549c6fd3a4443743eca99c6c3fe0e
Top:    dfc82f4d53ea5535aed220e5885345b2bc56e6d6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-08 00:57:52 +0000

Refresh of pcb-gl-fix-gl-headers-for

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 2048fef..f0db802 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -32,12 +32,18 @@
 #include <math.h>
 #include <assert.h>
 
-/* The Linux OpenGL ABI 1.0 spec requires that we define
- * GL_GLEXT_PROTOTYPES before including gl.h or glx.h for extensions
- * in order to get prototypes:
- *   http://www.opengl.org/registry/ABI/
- */
-#define GL_GLEXT_PROTOTYPES 1
+#ifdef WIN32
+#   define WIN32_LEAN_AND_MEAN 1
+#   include <windows.h>
+#else
+    /* The Linux OpenGL ABI 1.0 spec requires that we define
+     * GL_GLEXT_PROTOTYPES before including gl.h or glx.h for extensions
+     * in order to get prototypes:
+     *   http://www.opengl.org/registry/ABI/
+     */
+#   define GL_GLEXT_PROTOTYPES 1
+#endif
+
 #ifdef HAVE_OPENGL_GL_H
 #   include <OpenGL/gl.h>
 #else
